src/components/modals/TradePreviewModalGP.vue

```diff
@@ -148,17 +148,17 @@
       </BalCard>
       <div v-if="trading.requiresApproval.value" class="flex justify-between">
         <BalBtn
-          v-if="!isUnlocked || approved"
+          v-if="!isApproved || approved"
```

Comment:
I guess `isApproved` is better, I just wanted to differentiate between "I have just approved" (`approved`) and "it was previously approved / just approved it" (`isApproved`)

Maybe what's confusing is `approved` and should be renamed to `recentlyApproved` or `justApproved`

Reply:
agree

Reply:
Yeah agree, instead I've just reverted back to `isUnlocked` we can change later if it makes sense. https://github.com/balancer-labs/frontend-v2/pull/570/commits/0bc070a09b2a3ff1aa74d638bebae8fa688e76ce